lib/Consumer.js

```diff
@@ -222,6 +223,14 @@ class Consumer extends EnhancedEventEmitter
 		// Remove notification subscriptions.
 		this._channel.removeAllListeners(this._internal.consumerId);
 
+		// Remove previously registered listener for transport's @close so that closed Consumer object can be picked up by GC
```

Comment:
MS3 no longer adds a transport even listener in consumer. This change can be discarded.